deeplabcut/pose_estimation_tensorflow/core/evaluate.py

```diff
@@ -18,6 +18,8 @@
 import pandas as pd
 from tqdm import tqdm
 
+import deeplabcut
```

Comment:
I'm not a fan of importing all of `deeplabcut` in the `core/evaluate` code... would it perhaps be better to just import the module/method that we need?

I believe that here it's just for the `return_train_network_path` method, which could be imported as 
```python
from deeplabcut.pose_estimation_tensorflow.training import return_train_network_path
```